### DISCUSSION TITLE
[suggestion] Explain the differences from Tusky

### DISCUSSION THREAD

**Discussion by K0-RR** [pachli/pachli-android#487] [Ideas]
Hello, I think in the readme some Tusky vs Pachli comparison would be nice.

**Comment by nikclayton** [comment]
Hi -- thanks for the suggestion.

At the moment I'm reluctant to do that, for the following reasons.

- It would rapidly get out of date. I threw together https://gist.github.com/nikclayton/71e33b892331a52ab0ed155d8c9d7a49 when someone asked for something a few months ago, but keeping it updated would be a moderate amount of work on a regular cadence (say, every two weeks).
- It could be perceived as this project denigrating the work of the Tusky project
- It's duplicating the work of https://docs.google.com/spreadsheets/d/1De5KRwqMIdwEryfoeBLARgxF7QgKkeOQBCilKuIdAXE/edit#gid=0 (although I don't know how up to date that spreadsheet is)
- I'd rather people get to the point of thinking of Pachli as its own thing, and not Tusky++

However, "reluctant" does not mean "no". So if you've got some ideas for how to do this in a way that mitigates those concerns (e.g., are you volunteering to manage a list of differences and keep it up to date?) then I'm very happy to hear them.
